.pre-commit-config.yaml

```diff
@@ -48,3 +48,8 @@ repos:
       language: system
       entry: shellcheck --severity=error -e SC2068
       types: [shell]
+-   repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.27.1
+    hooks:
+    - id: markdownlint
+      args: ['--fix', '--config', '.markdownlint.yaml', '--ignore', '.github']
```

Comment:
I don't think you need --config if you use json.
Also --ignore on .github is that right?

Reply:
The .github one is required as the pull_request_template.md isn't really a proper document; it has no title and some of the rules don't apply. If you _want_ to make it a proper document that's fine too.

Reply:
Btw, you can exclude directly in pre-commit, which will prevent pre-commit from trying to run the hook on those files.

https://github.com/pypa/build/blob/5a3c5243a2135236f207da3a4b6401e1561f952d/.pre-commit-config.yaml#L11

Reply:
oh, specific to markdownlint-cli? Got an example?

Reply:
We run on --all-files for CI. So I think the exclusion you're doing is fine